### ISSUE TITLE
quippy: Potential has no attribute calc

### ISSUE THREAD

**Issue by mcaroba** [libAtoms/QUIP#264]
Hi, I'm getting this error when importing `quippy`:

```
>>> import quippy
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/u/21/carom1/unix/apps/quip/quip_dev_old/build/linux_x86_64_gfortran/quippy/__init__.py", line 42, in <module>
    import quippy.potential
  File "/u/21/carom1/unix/apps/quip/quip_dev_old/build/linux_x86_64_gfortran/quippy/potential.py", line 43, in <module>
    class Potential(ase.calculators.calculator.Calculator):
  File "/u/21/carom1/unix/apps/quip/quip_dev_old/build/linux_x86_64_gfortran/quippy/potential.py", line 114, in Potential
    @set_doc(quippy.potential_module.Potential.calc.__doc__,
AttributeError: type object 'Potential' has no attribute 'calc'
```

I'm running the latest development versions of `f90wrap`, `QUIP` and `GAP`. Just to triple check, I tried with and without `TurboGAP` support and I also tried the latest version of `GAP` before the `turbosoap` branch was merged to master, but I'm still getting the same error.

Does anyone know what might be wrong?

**Comment by jameskermode** [comment]
I’ve also seen this, it’s a regression in f90wrap. I won’t have time to fix it until new year so please use the version tagged in QUIP rather than the latest version until then.

**Comment by mcaroba** [comment]
Thanks for letting me know! I will revert to the previous f90wrap version.

**Comment by jameskermode** [comment]
This turned out to be quite a complex regression in f90wrap, so thanks for drawing it my attention. Should be fixed once jameskermode/f90wrap#136 is merged, so I'll close this issue, but do re-open if the problem recurs.

**Comment by mcaroba** [comment]
Thanks James for keeping always right on top of all the issues!
